src/renderer/components/wallet/settings/Settings.stories.tsx

```diff
@@ -17,8 +16,12 @@ storiesOf('Wallet/Settings', module).add('default', () => {
       clientUrl={some(BNB_ADDRESS_TESTNET)}
       lockWallet={() => console.log('lock')}
       removeKeystore={() => console.log('removeKeystore')}
-      retrieveLedgerAddress={({ chain }: LedgerAddressParams) => console.log('retrieve ledger address: ', chain)}
-      removeLedgerAddress={(chain: Chain) => console.log('remove ledger address: ', chain)}
+      retrieveLedgerAddress={({ chain, network }: LedgerAddressParams) =>
+        console.log('retrieve ledger address: ', chain, network)
+      }
+      removeLedgerAddress={({ chain, network }: LedgerAddressParams) =>
+        console.log('remove ledger address: ', chain, network)
+      }
```

Comment:
Do we need to pass network around to remove ledger? Changing from one network to another, the previous connected account should be removed.

Reply:
@veado hm. why changing network should disconnect connected address ?

Reply:
IMO it does not make sense to store an address for testnet in memory if you running mainnet for example.

Reply:
@veado but it makes sense when you switch between networks. main => test => main => etc. 
about storing few additional objects in a memory when we are talking about JS.... :) im not sure that few objects in a memory is really a thing which worth to worry about :)

Reply:
Can we keep it simple for now in this PR and continue discussion again later if this topic is raised as important one?

Reply:
We never store any data of another network. It's not about memory, it just does not make sense. There is a very rare use case somebody would switch between networks. You will use ASGDX to do swap, deposit, send, receive, but not to toggle between networks at first place.

Reply:
@veado oooookay. if we dont need to care about re-connecting to the device after switching between networks that single stream is enough here.